tools/sklearn/search_plot.xml

```diff
@@ -0,0 +1,87 @@
+<tool id="sk_search_plot" name="Plot Visualization" version="@VERSION@">
+    <description>of SearchCV results</description>
+    <macros>
+        <import>main_macros.xml</import>
```

Comment:
I think this tool should have its own folder. It is independent of sklearn, isn't it? At least it should be.
This means we can also get rid of the macro imho, we do not use it to a large degree isn't it?